package.json

```diff
@@ -40,6 +40,7 @@
     "@types/inquirer": "9.0.3",
     "@typescript-eslint/eslint-plugin": "^6.7.3",
     "@typescript-eslint/parser": "^6.7.3",
+    "browserify-zlib": "^0.2.0",
```

Comment:
Recommended change from `yarn`

Reply:
recommended how? not seeing anything related to this and have seen no indication it is needed. hdwallet is already huge, please remove at this time

Reply:
Hmm I probably meant to add this to sandbox only. It's related to resolving this warning when running `yarn dev:sandbox`, but regardless should probably belong in a different PR anyway
```
▶ yarn dev:sandbox
yarn run v1.22.19
$ cd examples/sandbox && yarn dev
$ yarn parcel index.html
$ /Users/millsmcilroy/workspace/hdwallet/node_modules/.bin/parcel index.html
Server running at http://localhost:1234

console: Browserslist: caniuse-lite is outdated. Please run:
npx browserslist@latest --update-db
Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
```

Will remove 👍

Reply:
ok, yeah that makes more sense. I would say for next time in general, it would be nice to separate the feature logic from the cleanup logic just to make reviews a bit more streamlined. thanks for taking the time though regardless!

Reply:
Parcel seems to be auto-installing `browserify-zlib` even in CI
<img width="1202" alt="Screenshot 2023-10-26 at 20 00 00" src="https://github.com/shapeshift/hdwallet/assets/37815163/2aadd3d0-65fc-4b28-a5c3-23e89fede952">

Reply:
No idea why, going to look at this again tomorrow with fresh eyes